src/mscorlib/model.xml

```diff
@@ -8152,6 +8152,10 @@
       <Member MemberType="Property" Name="Local" />
       <Member MemberType="Property" Name="Utc" />
     </Type>
+    <Type Name="System.TimeZoneInfo+AdjustmentRule">
+      <!-- CreateAdjustmentRule doesn't have a public contract, but tests and other callers can use Reflection to invoke it, so ensure it is left in mscorlib. -->
```

Comment:
Looking at the mscorlib.dll from before the change (see \\cpvsbuild\Drops\dev14\ProjectKRel\raw\23117.00\binaries.amd64ret\ProjectK\Runtime\mscorlib.dll), the method was public, and the class was internal.  From JustDecompile:

```C#
internal sealed class AdjustmentRule : IEquatable<TimeZoneInfo.AdjustmentRule>
{
  ...
  public static TimeZoneInfo.AdjustmentRule CreateAdjustmentRule(DateTime dateStart, DateTime dateEnd, TimeSpan daylightDelta, TimeZoneInfo.TransitionTime daylightTransitionStart, TimeZoneInfo.TransitionTime daylightTransitionEnd)
            {
```